Symfony/CS/ConfigurationException/InvalidConfigurationException.php

```diff
@@ -24,9 +24,10 @@ class InvalidConfigurationException extends \InvalidArgumentException
 {
     /**
      * @param string $message
+     * @param int    $code
```

Comment:
It is general style of default values - which in this one case indeed gives you nothing but consistency.

General issue:
`function foo($a, $b = 1, $c = 2) {}`
vs
`function foo($a, $b = null, $c = null) {}`

Try to pass $a and $c, but leave the $b as default in first example. You must know the default, and when it changes you need to change calls. In second example you just pass null. Everything would be cool if `default` keyword would be introduced, but it is not.
Also, that strategy of dealing is also used by native PHP functions.